tools/configen/configen/configen.py

```diff
@@ -112,9 +114,7 @@ def is_incompatible(type_: Type[Any]) -> bool:
         try:
             OmegaConf.structured(type_)  # verify it's actually legal
         except ValidationError as e:
-            log.debug(
-                f"Failed to create DictConfig from ({type_.__name__}) : {e}, flagging as incompatible"
-            )
+            log.debug(f"Failed to create DictConfig from ({type_.__name__}) : {e}, flagging as incompatible")
```

Comment:
Please use the same buck configuration as Hydra. Read the developer guide to see how to run the linters locally with nox.

Reply:
yeah, we have a slightly different black configuration in NeMo (voted for longed lines, 120 chars).
So I have integrated that on the level of editor (VSCode), now need to figure it out how to differentiate that for different projects...